src/domain/orders/details/returns/receive-menu.tsx

```diff
@@ -7,6 +7,7 @@ import CurrencyInput from "../../../../components/organisms/currency-input"
 import RMASelectProductTable from "../../../../components/organisms/rma-select-product-table"
 import { getErrorMessage } from "../../../../utils/error-messages"
 import { displayAmount } from "../../../../utils/prices"
+import RMASelectReturnProductTable from "../../../../components/organisms/rma-select-receive-product-table"
 
 type ReceiveMenuProps = {
   order: any
```

Comment:
Since we are updating this I think we should move away from not typing it correctly 😄

Reply:
We can also remove `isSwapOrClaim` since it's not used